tests/.eslintrc.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'Factory': false,
```

Comment:
I'm loathe to use magical globals that are not declared anywhere; for example, the long-term plan for ember's test helpers (ie `click`, `fillIn`, etc) is to have them be imports rather than globals. If we can't import it, it seems better to just declare it as a matter of course, so that it's obviously being used in that file. The linting on it is actually quite nice; if you use a global without declaring it via comment, it throws an error, and if you remove the global altogether, the linting throws another error to say that you declared an unused global. That's super helpful for being confident about where you are using your globals.